README.md

```diff
@@ -74,7 +75,7 @@ Base 32: 14P_C0MI
 Enter `numconverter --help` for available options.
 
 ```
-numconverter [base_char] <from_num> [to_base]... [FLAGS]... [OPTIONS]...
+numconverter [base_char] [from_num] [to_base]... [FLAGS]... [OPTIONS]...
```

Comment:
Not really a standard, but I like to keep required arguments in `<>` brackets, optional arguments in `[]` brackets.

https://stackoverflow.com/a/36787811/8862937

Reply:
Whoops, obviously that argument is now optional due to the `--from-clipboard option`.   Sorry, commented before thinking about it!